src/test/java/org/opensearch/security/dlic/rest/api/ActionGroupsApiTest.java

```diff
@@ -28,201 +32,216 @@
 import org.opensearch.security.test.helper.file.FileHelper;
 import org.opensearch.security.test.helper.rest.RestHelper.HttpResponse;
 
+@RunWith(Parameterized.class)
 public class ActionGroupsApiTest extends AbstractRestApiUnitTest {
 
-	@Test
-	public void testActionGroupsApi() throws Exception {
+    private final String ENDPOINT;
 
-		setup();
+    public ActionGroupsApiTest(String endpoint){
+        ENDPOINT = endpoint;
+    }
+
+    @Parameterized.Parameters
+    public static Collection input() {
```

Comment:
use `Iterable<String>` instead of `Collection`. nit: rename `input()` to `endpoints()`.